CONTRIBUTING.md

```diff
@@ -142,21 +142,24 @@ tracing) enabled.
 The setup for a local rustc works as follows:
 ```sh
 # Clone the rust-lang/rust repo.
-git clone https://github.com/rust-lang/rust/ rustc
+git clone https://github.com/rust-lang/rust rustc
 cd rustc
-cp config.toml.example config.toml
-# Now edit `config.toml` and set `debug-assertions = true`.
+# Create a config.toml with defaults for working on miri
+./x.py setup compiler
```

Comment:
So this enables debug assertions, and you are making some form of guarantee that it will continue to do so?

Reply:
This enables debug logging, but not debug assertions; I misread what was there before. I still think this is good to suggest though, since if you have a config.toml `x.py` won't suggest running `setup`. How about this?
```suggestion
./x.py setup compiler
# Now edit `config.toml` and under `[rust]` set `debug-assertions = true`.
```

Reply:
Sure, that also works.